tests/integration-tests/tests/cloudwatch_logging/test_compute_console_output_logging.py

```diff
@@ -0,0 +1,159 @@
+import json
+import logging
+import re
+
+import boto3
+import pytest
+from assertpy import assert_that
+from clusters_factory import Cluster
+from configparser import ConfigParser
+from remote_command_executor import RemoteCommandExecutor
+from retrying import retry
+from time_utils import minutes
+
+from tests.cloudwatch_logging.cloudwatch_logging_boto3_utils import (
+    get_cluster_log_groups_from_boto3,
+    get_log_events,
+    get_log_streams,
+)
+
+logger = logging.getLogger(__name__)
+
+
+@pytest.fixture
+def test_resources_dir(datadir):
```

Comment:
Is test_resources_dir being used in the code?

Reply:
It is not. Removed.